src/app/containers/MediaPlayer/index.jsx

```diff
@@ -17,7 +17,9 @@ import {
   emptyBlockArrayDefaultProps,
 } from '#models/propTypes';
 
-const MediaPlayerContainer = ({ blocks, placeholder }) => {
+const MediaPlayerContainer = ({ blocks, placeholder, embedOverrides = {} }) => {
```

Comment:
Can you add `embedOverrides` to propTypes with a shape and a default please? Instead of defaulting in the func args